### PR TITLE
Update 0.153 release notes

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.153.rst [hunk 1]
```diff
@@ -6,32 +6,154 @@ General Changes
 ---------------
 
 * Fix incorrect results for grouping sets when ``task.concurrency`` is greater than one.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: We should probably add a warning in the release notes for the affected versions

File: presto-docs/src/main/sphinx/release/release-0.153.rst [hunk 2]
```diff
@@ -6,32 +6,154 @@ General Changes
...
+* Fix null handling in :func:`array_distinct` when applied to the ``array(bigint)`` type.
+* Fix handling of ``-2^63`` as the element index for :func:`json_array_get`.
+* Fix correctness issue when the input to ``TRY_CAST`` evaluates to null.
+  For types such as booleans, numbers, dates, timestamps, etc., rather than
+  returning null, a default value specific to the type such as
+  ``false``, ``0`` or ``1970-01-01`` was returned.
+* Fix potential thread deadlock in coordinator.
+* Fix rare correctness issue with an aggregation on a single threaded right join when
+  ``task.concurrency`` is ``1``.
+* Fix query failure when casting a map with null elements in values.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: "... casting a map with null values". "elements in" is redundant.

File: presto-docs/src/main/sphinx/release/release-0.153.rst [hunk 3]
```diff
@@ -6,32 +6,154 @@ General Changes
...
+* Fix failure when view column names contain upper-case letters.
+* Fix potential performance regression due to skew issue when
+  grouping or joining on columns of the following types: ``TINYINT``,
+  ``SMALLINT``, ``INTEGER``, ``BIGINT``, ``REAL``, ``DOUBLE``,
+  ``COLOR``, ``DATE``, ``INTERVAL``, ``TIME``, ``TIMESTAMP``.
+* Fix potential memory leak for delete queries.
+* Fix query stats to not include queued time in planning time.
+* Fix spurious log messages when queries are torn down.
+* Remove broken ``%w`` specifier for :func:`date_format` and :func:`date_parse`.
+* Make resource group configuration more flexible.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: This should go in the new features section.

File: presto-docs/src/main/sphinx/release/release-0.153.rst [hunk 4]
```diff
@@ -6,32 +6,154 @@ General Changes
...
+* Add decimal support to :func:`avg`, :func:`ceil`, :func:`floor`, :func:`round`,
+  :func:`truncate`, :func:`abs`, :func:`mod` and :func:`sign`.
+* Add :func:`shuffle` function for arrays.
 * Add :doc:`/connector/accumulo`.
 
-SPI Changes
+Web UI Changes
+--------------
+
+* Fix rendering failures in due to null nested data structures.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: "rendering failures in ??? due to..."

File: presto-docs/src/main/sphinx/release/release-0.153.rst [hunk 5]
```diff
@@ -6,32 +6,154 @@ General Changes
...
+* Fix active worker count on cluster overview page to not include coordinator.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: Maybe rephrase as "Remove coordinator from active worker count in cluster overview page."

File: presto-docs/src/main/sphinx/release/release-0.153.rst [hunk 6]
```diff
@@ -6,32 +6,154 @@ General Changes
...
+* Replace buffer skew indicators on query details page with scheduled time skew.
+* Add stage total buffer, pending tasks and wall time to stage statistics on query details page.
+* Add option to filter task lists by status on query details page.
+* Add copy button for query text, query ID, and user to query details page.
+
+JDBC Driver Changes
+-------------------
+
+* Add support for ``REAL`` data type which corresponds to the Java ``float`` type.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: Missing comma: "... data type, which corresponds ..."

File: presto-docs/src/main/sphinx/release/release-0.153.rst [hunk 7]
```diff
@@ -6,32 +6,154 @@ General Changes
...
+
+CLI Changes
 -----------
 
-* Add support for pluggable resource group management. A ``Plugin`` can now
-  provide management factories via ``getResourceGroupConfigurationManagerFactories()``
-  and the factory can be enabled via the ``etc/resource-groups.properties``
-  configuration file by setting the ``resource-groups.configuration-manager``
-  property. See the ``presto-resource-group-managers`` plugin for an example.
+* Add support for configuring the HTTP Truststore.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: https? ssl?

File: presto-docs/src/main/sphinx/release/release-0.153.rst [hunk 8]
```diff
@@ -6,32 +6,154 @@ General Changes
...
 
 Hive Changes
 ------------
 
+* Fix permissions for new tables when using SQL-standard authorization.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: Describe what was the issue that was fixed.

File: presto-docs/src/main/sphinx/release/release-0.153.rst [hunk 9]
```diff
@@ -6,32 +6,154 @@ General Changes
...
+  for example, when dropping a partition and then recreating it. Atomicity is not
+  guaranteed due to fundamental limitations in the design of Hive.
+* Fail queries that attempt to rename partition columns.
+* Add support for ORC bloom filters in predicate push down.
+  This is can be enabled using the ``hive.orc.bloom-filters.enabled``
+  configuration property or the ``orc_bloom_filters_enabled`` session property.
+* Add new optimized RCFile reader.
+  This can be enabled using the ``hive.rcfile-optimized-reader.enabled``
+  configuration property or the ``rcfile_optimized_reader_enabled`` session property.
+* Add support for the Presto ``REAL`` type which corresponds to the Hive ``FLOAT`` type.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: missing comma before "which"

File: presto-docs/src/main/sphinx/release/release-0.153.rst [hunk 10]
```diff
@@ -6,32 +6,154 @@ General Changes
...
+* Add support for creating, dropping and renaming schemas (databases).
+  The filesystem location can be specified when creating a schema,
+  which allows, for example, easily creating tables on S3.
+* Record Presto query ID for tables or partitions written by Presto
+  using the ``presto_query_id`` table or partition property.
+* Support per-transaction cache for Hive metastore.
+* Include path name in error message when listing a directory fails.
+* Rename ``allow-all`` authorization method to ``legacy``. This
+  method is deprecated and will be removed in a future release.
+* Do not retry intentionally-aborted S3 requests.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: "intentionally-aborted" sounds weird. I'd rephrase it as "Do not retry S3 requests that are aborted intentionally".

File: presto-docs/src/main/sphinx/release/release-0.153.rst [hunk 11]
```diff
@@ -6,32 +6,154 @@ General Changes
...
 
-* Add support for ``char(x)`` type.
+* Make name configuration for history tables case-insensitive.
 
 MySQL Changes
 -------------
 
-* Add support for ``char(x)`` type.
+* Optimize fetching column names when describing a single table.
+* Add support for ``char(x)`` and ``REAL`` data types.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: capitalize "char" or lower-case "real". Also, name it "char(n)", not "char(x)" -- it's more commonly used that way.

File: presto-docs/src/main/sphinx/release/release-0.153.rst [hunk 12]
```diff
@@ -6,32 +6,154 @@ General Changes
...
+
+PostgreSQL Changes
+------------------
+
+* Optimize fetching column names when describing a single table.
+* Add support for ``char(x)`` and ``REAL`` data types.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: Same as above.

File: presto-docs/src/main/sphinx/release/release-0.153.rst [hunk 13]
```diff
@@ -6,32 +6,154 @@ General Changes
...
+-----------
+
+* Add support for pluggable resource group management. A ``Plugin`` can now
+  provide management factories via ``getResourceGroupConfigurationManagerFactories()``
+  and the factory can be enabled via the ``etc/resource-groups.properties``
+  configuration file by setting the ``resource-groups.configuration-manager``
+  property. See the ``presto-resource-group-managers`` plugin for an example.
+* Add ``schemaExists()`` method to ``ConnectorMetadata``.
+* Add transaction to grant/revoke in ``ConnectorAccessControl``.
+* Add ``isCoordinator()`` and ``getVersion()`` methos to ``Node``.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: typo: methos

File: presto-docs/src/main/sphinx/release/release-0.153.rst (hunk 1 above)
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: Which ones are affected?

Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: All releases after that property was added, I think. @raghavsethi?

File: presto-docs/src/main/sphinx/release/release-0.153.rst (hunk 4 above)
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: Forgot to remove "in" when I removed "Web UI" (since it's in the UI section)

File: presto-docs/src/main/sphinx/release/release-0.153.rst (hunk 5 above)
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: Changed to "Do not include coordinator in active worker count on cluster overview page."

File: presto-docs/src/main/sphinx/release/release-0.153.rst (hunk 11 above)
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: Updated everywhere to lowercase since we normally lowercase type names

File: presto-docs/src/main/sphinx/release/release-0.153.rst (hunk 1 above)
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.153.rst]: I'm not sure we should update everything. There are other bugs like `try_cast` that would require adding warnings to dozens of releases. I think it's understood that each release can fix correctness bugs, and if you continue to run an old release, you should see what is fixed in newer releases.
